msteams-platform/bots/what-are-bots.md

```diff
@@ -14,11 +14,9 @@ It's recommended to start with [build your first bot app using JavaScript](../sb
 
 > [!IMPORTANT]
 >
-> * Bots are available in Government Community Cloud (GCC) and GCC-High but not in Department of Defense (DOD).
+> * Bots are available in [Government Community Cloud (GCC) and Department of Defense (DOD)](~/concepts/app-fundamentals-overview.md#government-community-cloud) environments. Bot applications within Microsoft Teams for GCC-High and DOD are available through [Azure bot Service](/azure/bot-service/how-to-deploy-gov-cloud-high) and bot channel registration must be done in Azure Government portal.
 >
-> * Bot applications within Microsoft Teams are available in GCC-High through [Azure bot Service](/azure/bot-service/how-to-deploy-gov-cloud-high) and bot channel registration must be done in Azure Government portal.
->
-> * Applications in GCCH only support up to manifest version v1.10. Image URLs in Adaptive Cards aren't supported in GCCH environment. You can replace an image URL with Base64 encoded DataUri.
+> * Applications in GCC-High only support up to manifest version v1.10. Image URLs in Adaptive Cards aren't supported in GCC-High and DOD environments. You can replace an image URL with Base64 encoded DataUri.
```

Comment:
I think we no longer have this limitation.

Reply:
The manifest version limitation no longer exists. Let's remove that.